bindings/wasm/src/account/storage/traits.rs

```diff
@@ -58,6 +54,8 @@ extern "C" {
   pub type PromiseEncryptedData;
   #[wasm_bindgen(typescript_type = "Promise<Uint8Array>")]
   pub type PromiseData;
+  #[wasm_bindgen(typescript_type = "Promise<Uint8Array | undefined>")]
+  pub type PromiseBytes;
```

Comment:
Nit: maybe `PromiseOptionBytes` would be more consistent.

Also remove `PromiseOptionChainState` and `PromiseOptionDocument` in this file, which are now unused.